### PR TITLE
completed react-elements - Jay Ghil

### PR DESCRIPTION
**Pull request by jghil**
image

### CODE REVIEW

File: react-elements/src/index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+const firstReact = React.createElement(
+  'h1',
+  null,
+  'Hello, React!'
+);
+console.log(firstReact);
+
+const container = document.querySelector('#root');
+const root = ReactDOM.createRoot(container);
+root.render();
```
Comment (scbowler) [inline react-elements/src/index.js]: You need to pass the element to `render` so it is displayed in the browser.
```JAVASCRIPT
root.render(firstReact);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but 1 update needed. Let me know if you have any questions.

File: react-elements/src/index.js (hunk 1 above)
Comment (jghil) [reply react-elements/src/index.js]: gotcha! that makes sense! thanks Scott!

Comment (jghil) [reply react-elements/src/index.js]: image

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Pull request merged**